examples/tv-casting-app/linux/main.cpp

```diff
@@ -235,7 +234,7 @@ void DeviceEventCallback(const DeviceLayer::ChipDeviceEvent * event, intptr_t ar
             .clientPool     = &gCASEClientPool,
         };
 
-        PeerId peerID = fabric->GetPeerIdForNode(tvNodeId);
+        PeerId peerID = fabric->GetPeerIdForNode(event->CommissioningComplete.PeerNodeId);
```

Comment:
@yufengwangca This code is not checking status, so will be reading random data here on failure, no?

Reply:
The PeerNodeID is put into the event itself, along with status. It always get the PeerNodeID which assigned when the event is created
CheckSuccess(server->CommissioningComplete(handle->AsSecureSession()->GetPeerNodeId()), Failure);

Reply:
Yes, the point is the consumer is not checking the status, so will try to read it even if nothing was written to the event and the data in the event is random....